src/pages/library/WorkflowsLibrary.tsx

```diff
@@ -0,0 +1,232 @@
+          lineHeight: '18px',
+          boxShadow: `3px 3px ${colors.grey(0.2)}`,
+        }}
+      >
+        <div style={{ marginLeft: '10px', marginTop: '4px' }}>
+          <LogoTile logoFilePath={props.logoFilePath} />
+          <div style={{ fontSize: '1rem', fontWeight: 600, color: colors.accent(1.1), marginTop: '5px' }}>
+            {props.title}
+          </div>
+          <p style={{ height: '55px' }}>{props.description}</p>
+          <div style={{ bottom: 0 }}>
+            {icon('pop-out', {
+              size: 18,
+              style: { color: colors.accent(1) },
+            })}
+          </div>
+        </div>
+      </div>
+    </Clickable>
+  );
+};
+
+const CuratedWorkflowsSection = () => {
+  const dockstoreUrlRoot: string = getConfig().dockstoreUrlRoot;
+
+  return (
+    <div>
+      <div style={{ marginTop: '25px' }}>
+        <p style={{ fontSize: '1rem', fontWeight: 600 }}>Curated collections from our community:</p>
+      </div>
+      <div
+        style={{
+          display: 'flex',
+          flexWrap: 'wrap',
+          flexDirection: 'row',
+          height: '100%',
+          position: 'relative',
+        }}
+      >
+        {_.map((wf: CuratedWorkflowDetails) => {
+          return (
+            <Clickable
+              key={wf.key}
+              href={wf.url}
+              {...Utils.newTabLinkProps}
+              style={{
+                ...Style.elements.card.container,
+                width: 150,
+                height: 110,
+                margin: '0 1rem 2rem 0',
+                color: colors.accent(1),
+              }}
+            >
+              <div>{wf.title}</div>
+              <div style={{ bottom: 0 }}>{icon('pop-out', { size: 14, style: { marginLeft: '0.25rem' } })}</div>
+            </Clickable>
+          );
+        }, curatedWorkflowsList(dockstoreUrlRoot))}
+      </div>
+    </div>
+  );
+};
+
+export const WorkflowsLibrary = () => {
+  const dockstoreUrl = `${getConfig().dockstoreUrlRoot}/search?_type=workflow&descriptorType=WDL&searchMode=files`;
+  const workflowsRepoUrl: string = isFeaturePreviewEnabled(FIRECLOUD_UI_MIGRATION)
+    ? Nav.getLink('workflows')
+    : `${getConfig().firecloudUrlRoot}/?return=${getEnabledBrand().queryName}#methods`;
+
+  return (
+    <FooterWrapper alwaysShow>
+      {libraryTopMatter('workflows')}
+      <main style={{ flexGrow: 1 }}>
+        <div style={{ display: 'flex', flexDirection: 'row', height: '100%', flex: 1 }}>
+          <div style={{ width: '80%', display: 'flex' }}>
+            <div style={{ flex: 1, margin: '30px 0 30px 40px' }}>
+              <div style={styles.header}>Discover Workflows</div>
+              <div style={{ display: 'flex', marginTop: '0.25rem' }}>
+                <div>
+                  <WorkflowSourceBox
+                    title='Dockstore.org'
+                    description='A community repository of public workflows that offers automatic integration with GitHub and publishing features.'
```

Comment:
Nit: I think this would sound more natural if we reversed the order of the things we offer:
> A community repository of public workflows that offers publishing features and automatic integration with GitHub.

The way it is now, my brain needs to take a second to figure out whether Dockstore offers "integration with ... publishing features."

Reply:
That makes sense to me. I have updated the wording.